### DISCUSSION TITLE
I have a problem with adapter in airckrack.

### DISCUSSION THREAD

**Discussion by nqm322** [aircrack-ng/aircrack-ng#2596] [Q&A]
I have wlan adapter - Intel(R) Dual Band Wireless-AC 8265, is this are supporting monitore mode in aircrack? Or which drivers i need to put, help please...

**Comment by Mister-X-** [comment]
Yes it works with any recent distro. Drivers are in the kernel so no need to install anything.

**Reply by nqm322** [reply]
But when i typing command iwconfig or ifconfig inside a aircrack terminal , i use virtualbox, machine are see my adapter but its no wirelles extensions , what i should do?

**Reply by Mister-X-** [reply]
That's because it is attached to the host, not to the guest (VM), so it doesn't see it. If you want to stick with that adapter, you'll need to do PCI passthrough, which is AFAIK only available on when Linux is used as a host: https://docs.oracle.com/en/virtualization/virtualbox/6.0/admin/pcipassthrough.html

And there is some boot configuration needed as well.

Otherwise, you'll need to use a USB adapter.
